test/CreatorToken.t.sol

```diff
@@ -484,3 +497,51 @@ contract Pausing is CreatorTokenTest {
     vm.stopPrank();
   }
 }
+
+contract bondingCurveTests is CreatorTokenTest {
+  function test_BuyPriceIsCorrect(address _buyer) public {
+    vm.assume(
+      _buyer != address(0) && _buyer != address(creatorToken) && _buyer != creator
+        && _buyer != admin
+    );
+    uint256 buyNTokens = 5;
```

Comment:
Instead of hardcoding this, we can make this fuzzable. i.e. add parameter like `_numTokensToBuy`, then use `bound` to put that at some reasonable upper limit, maybe 10K for example.

Reply:
upper limit of 100 takes 3.5s on my machine, slowing the whole `forge test`to 7.52s. I think 100 is sufficient, but if you disagree we can up it more.

Reply:
Upper limit of 100 here: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/6a0ad6858e103fa6da6384492f59571f85783ba2